helix-query.yaml

```diff
@@ -69,86 +69,56 @@ indices:
 
   italy:
     <<: *default
     include:
-      - 'it-IT/(make|templates)/**/*.(docx|md)'
-    target: https://adobe.sharepoint.com/:x:/r/sites/SparkHelix/Shared%20Documents/website/it-IT/query-index.xlsx?d=w96cd087a998c4398b64f7a0fd02f9d23&csf=1&web=1&e=bzFIT6
+      - 'it/express/**/*.(docx|md)'
+    target: https://adobe.sharepoint.com/:x:/r/sites/SparkHelix/Shared%20Documents/website/it/express/query-index.xlsx?d=wf5fe76ac74c84137bc971ddb26c4c025&csf=1&web=1&e=iCAPc1
 
   japan:
     <<: *default
     include:
-      - 'ja-JP/(make|templates)/**/*.(docx|md)'
-    target: https://adobe.sharepoint.com/:x:/r/sites/SparkHelix/Shared%20Documents/website/ja-JP/query-index.xlsx?d=w4a9640dc295b436198e93cd41dda88f0&csf=1&web=1&e=dsJW31
+      - 'jp/express/**/*.(docx|md)'
+    target: https://adobe.sharepoint.com/:x:/r/sites/SparkHelix/Shared%20Documents/website/jp/express/query-index.xlsx?d=wbb195af834f14eb893a69fb45c8647b7&csf=1&web=1&e=7HQUIf
 
   korea:
     <<: *default
     include:
-      - 'ko-KR/(make|templates)/**/*.(docx|md)'
-    target: https://adobe.sharepoint.com/:x:/r/sites/SparkHelix/Shared%20Documents/website/ko-KR/query-index.xlsx?d=w5f70bfa2846f4b32bd494515f16a1a1c&csf=1&web=1&e=5BYelj
-
-  norway:
-    <<: *default
-    include:
-      - 'nb-NO/(make|templates)/**/*.(docx|md)'
-    target: https://adobe.sharepoint.com/:x:/r/sites/SparkHelix/Shared%20Documents/website/nb-NO/query-index.xlsx?d=w91e0ce70cc8b4eb9b96f998357678df1&csf=1&web=1&e=RoQbEY
+      - 'kr/express/**/*.(docx|md)'
+    target: https://adobe.sharepoint.com/:x:/r/sites/SparkHelix/Shared%20Documents/website/kr/express/query-index.xlsx?d=w6eb6c21b8313419ab341631c55901a7d&csf=1&web=1&e=3m9As0
 
   netherlands:
     <<: *default
     include:
-      - 'nl-NL/(make|templates)/**/*.(docx|md)'
-    target: https://adobe.sharepoint.com/:x:/r/sites/SparkHelix/Shared%20Documents/website/nl-NL/query-index.xlsx?d=wf708becba51b46b2a8ee22d0fc2d1235&csf=1&web=1&e=didBoW
+      - 'nl/express/**/*.(docx|md)'
+    target: https://adobe.sharepoint.com/:x:/r/sites/SparkHelix/Shared%20Documents/website/nl/express/query-index.xlsx?d=wfd7830086887453aa33a6567e3bbea21&csf=1&web=1&e=3eJ9xU
 
   brasil:
     <<: *default
     include:
-      - 'pt-BR/(make|templates)/**/*.(docx|md)'
-    target: https://adobe.sharepoint.com/:x:/r/sites/SparkHelix/Shared%20Documents/website/pt-BR/query-index.xlsx?d=w3dbeff4403034f3a9bc03dfb28c8da65&csf=1&web=1&e=1VpwLM
-
-  sweden:
-    <<: *default
-    include:
-      - 'sv-SE/(make|templates)/**/*.(docx|md)'
-    target: https://adobe.sharepoint.com/:x:/r/sites/SparkHelix/Shared%20Documents/website/sv-SE/query-index.xlsx?d=w34788cef298e4153a53114321d90cadd&csf=1&web=1&e=dGRvwC
-
-  china:
-    <<: *default
-    include:
-      - 'zh-Hans-CN/(make|templates)/**/*.(docx|md)'
-    target: https://adobe.sharepoint.com/:x:/r/sites/SparkHelix/Shared%20Documents/website/zh-Hans-CN/query-index.xlsx?d=w217d1db4ba2847278e5eaa3bae6dbb4e&csf=1&web=1&e=vkTQAw
```

Comment:
swedish and chinese site are no longer required, I assume?

Reply:
Yes, they cleaned up. I will raise the feedback and might have to change that later again.